discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -130,20 +133,13 @@ def select_installation_disk(self, hosts_with_disk_paths):
 
     def set_ocs(self, ocs_enabled):
         logging.info(f'Enabling Ocs to:{ocs_enabled} for cluster: {self.id}')
```

Comment:
```suggestion
        logging.info(Setting Ocs to:{ocs_enabled} for cluster: {self.id}')
```

Reply:
The code is in downstream repo. I apply your suggestion but I need to make correct string. You forgot `'`